src/caret_analyze/infra/lttng/ros2_tracing/processor.py

```diff
@@ -64,34 +67,16 @@ def __init__(
         ----------
         data : Ros2DataModel
             DataModel to be handles
+        remapper : IDRemapperCollection
+            Collections of IDRemapper.
```

Comment:
Minor fix of comment.
```suggestion
            Collection of IDRemappers.
```